streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -1127,6 +1175,36 @@ public void shouldUpdateInputPartitionsOfTasksRemovedFromStateUpdater() {
         Mockito.verify(stateUpdater).add(task01);
     }
 
+    @Test
+    public void shouldCloseReviveAndUpdateInputPartitionsOfTasksRemovedFromStateUpdater() {
+        final StreamTask task00 = statefulTask(taskId00, taskId00ChangelogPartitions)
```

Comment:
nit:
Could you please use more meaningful names for those task variables? Something like `activeTask` and `standbyTask` would already be better.

Reply:
Done